ntbs-service/Services/SpecimenService.cs

```diff
@@ -2,20 +2,22 @@
 using System.Threading.Tasks;
 using Dapper;
 using Microsoft.Extensions.Configuration;
-using System.Linq;
 using System.Collections.Generic;
 using ntbs_service.Models.Entities;
+using System.Data;
 
 namespace ntbs_service.Services
 {
     public interface ISpecimenService
     {
         Task<IEnumerable<Specimen>> GetSpecimenDetailsAsync(int notificationId);
+        Task UnmatchSpecimen(int notificationId, string labReferenceNumber);
     }
 
     public class SpecimenService : ISpecimenService
     {
-        private readonly string connectionString;
+        private readonly string reportingDbConnectionString;
+        private readonly string matchSpecimenDbConnectionString;

```

Comment:
Might be better to have this descriptive of the db rather than what we want to do with it. (Though I appreciate this one is `specimen-matching` I think a small change to `specimenMatchingDbConnectionString` might be nice).